docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -97,3 +97,15 @@ delay before consuming or producing a new page::
       page_processing_delay = '5s'
     );
 
+.. _blackhole-sql-support:
+
+SQL support
+-----------
+
+The connector provides access to data temporarily stored in the Black Hole
+storage engine. The connector supports the following features:
+
+* :doc:`/sql/insert`
+
+Supported SQL statements are logged as binary and only when binary logging is
+enabled.
```

Comment:
what does this mean?

Reply:
@findepi From what I understand, Black Hole stores binary logs and SQL statements are logged when binary logging is enabled.

Reply:
I think the confusion is coming due to the fact that MySQL also has a blackhole engine - https://dev.mysql.com/doc/refman/8.0/en/blackhole-storage-engine.html

Both are unrelated.

Reply:
I can't tell which experts are referring to MySQL and which ones aren't. They don't mention their sources. I'm dumping this sentence.